Sources/MapboxMaps/Foundation/MapView.swift

```diff
@@ -69,6 +69,8 @@ open class MapView: UIView {
     private var displayCallback: (() -> Void)?
     @objc dynamic internal var displayLink: CADisplayLink?
 
+    private var pixelRatio: CGFloat = 3.0 /// Same default value as found in `MapInitOptions`
```

Comment:
Could it be `private var pixelRatio: CGFloat = UIScreen.main.scale` or simply `0.0`? The `pixelRatio` will be always overwritten by `self.pixelRatio = CGFloat(resolvedMapInitOptions.mapOptions.pixelRatio)`, right?

Reply:
No, a user can initialize a MapView without the `MapOptions` so we need to provide a default

Reply:
> No, a user can initialize a MapView without the MapOptions

Could you please tell what is the codepath where `self.pixelRatio = CGFloat(resolvedMapInitOptions.mapOptions.pixelRatio)` will not be executed for constructed MapView and default will be used?

Reply:
You are correct, I was just checking it and it will always be initialized by the init. 

I thought we put it inside an if condition